cdm/src/test/java/ucar/nc2/ft/TestCFSimpleGeometryCont.java

```diff
@@ -0,0 +1,316 @@
+package ucar.nc2.ft2.coverage.simpgeometry;

```

Comment:
Should these tests be in separate classes?

Reply:
Perhaps but most tests are in ucar.nc2! Could revisit later.

Reply:
They will be split in the end, thanks!